src/components/Greeting.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getGreeting } from '../redux/Greetings/greetingsSlice';
+
+const Greeting = () => {
+  const greeting = useSelector((state) => state.greetings);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getGreeting());
+  });
+
+  return (
```

Comment:
- [Optional ] Please add the dependency array here so the `useEffect` run only when the user reloads the page, as here the `useEffect` is calling the function an unlimited number of times it should look something like this
![image](https://user-images.githubusercontent.com/74744787/216792576-4a54130e-c43b-4301-aed6-dd0cd9afbd51.png)